ElectronNET.CLI/ElectronNET.CLI.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
 
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Is moving to net6 required for this?  doing this move would cause a major version upgrade.  I believe that we shouldn't combine the 2 changes into a single pr here.

Reply:
Yes and no. In order to compile the tool for native distribution on M1 it is required. However, to compile an application for M1, the tool itself could use .net 5 but .net 6 would also need to be installed. 

I have debated between changing it or not and realize that it is a bit out of the scope of one pr, but I decided to add it and get input on it for now. If i didn't bump the version, at a minimum I would like to add a notification to the user that .net6 is required for arm Mac if they don't have it installed already. Im still looking into a way to do that.

Reply:
[https://docs.microsoft.com/en-us/dotnet/core/install/macos#arm-based-macs](https://docs.microsoft.com/en-us/dotnet/core/install/macos#arm-based-macs)

Reply:
Thanks for the info, i'll have to give that a try.

Reply:
I backed the version down to dotnet 5 and added a check to see if 6 is installed. Thanks for the tip!